src/renderer/components/input/input.validators.ts

```diff
@@ -44,7 +44,7 @@ export const isUrl: Validator = {
 
 export const isPath: Validator = {
   condition: ({ type }) => type === "text",
-  message: () => _i18n._(t`This field must be a path to an existing file`),
+  message: () => _i18n._(t`This field must be a path to an existing folder`),
```

Comment:
In general it could be either a file or folder? What about `This field must be a valid path` or `This field must be a valid file system path`?

Reply:
I was about to suggest similar.

Reply:
Or could have a folder validator and a file validator?

Reply:
@jim-docker `This field must be a valid path` makes totally sense, thanks!

Reply:
Boy, I miss CodeCollaborator (I think that's the one) where you could open a chat at any line instead of just adding a static comment. It would've avoided us all composing the same answer  :)